pepper-apis/studybuilder-cli/studies/singular/i18n/en.conf

```diff
@@ -1599,6 +1599,143 @@
+        "other_physician_details": "Select speciality",
+
+        "name": "PHYSICIAN NAME:",
+        "institution": "HOSPITAL, CLINIC, OR PROVIDER NAME (if any):",
+        "address": "ADDRESS:",
+        "phone": "PHONE NUMBER:",
+        "fax": "FAX NUMBER:",
+        "add_button": "CLICK HERE TO ADD MORE PHYSICIANS OR PROVIDERS",
+        "prompt": "Physician",
+        "additional_title": "Other Physician",
+
+        "validations": {
+          "required_field" : "Please fill required fields"
+        },
+
+
+        "upload": {
+          "p1": "For participants who are opting to send their own medical records to Project Singular study staff you may use one of the below methods.",
+          "li1": "Fax: (617)730-6201",
+          "li2": "Online Upload: If you have access to your own medical records files, you may upload the files via the Medical Record File Upload form available during enrollment in Project Singular and on the Project Singular dashboard."
+        }
+      },
+
+      "name": {
+        "prompt": "$ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"Parent/Guardian Name\", \"Name\")*",
+        "placeholder": "$ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"Parent/Guardian Name\", \"Your name\")",
+        "required_hint": "$ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"Please type in Parent/Guardian Name\", \"Please type in your name\")"
+      },
+      "dob": {
+        "prompt": "$ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"Your child's\", \"Patient\") date of birth:*",
+        "required_hint": "Please type in $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"your child’s\", \"your\") date of birth",
+        "format_hint": "Please type in $ddp.checkAnswer(\"MRR_WHO_ENROLLING_COPY\",\"CHILD\", \"your child’s\", \"your\") date of birth in MM DD YYYY format"
+      },
+      "date": {
+        "title": "Date"
+      },
+      "patient_name": {
+        "prompt": "Name of patient: *",
+        "placeholder": "Name of patient",
+        "required_hint": "Please type in name of patient"
+      },
+      "legal_dob": {
+        "prompt": "Date of birth: *",
+        "required_hint": "Please type in patient’s date of birth",
+        "format_hint": "Please type in patient’s date of birth in MM DD YYYY format"
+      },
+      "parent_name": {
+        "prompt": "Name of parent or legally authorized representative: *",
+        "placeholder": "Name of parent or legally authorized representative",
+        "required_hint": "Please type in name of parent or legally authorized representative"
+      },
+      "parent_signature": {
+        "prompt": "Signature of parent or legally authorized representative: *",
+        "placeholder": "Type parent or legally authorized full name to sign",
+        "required_hint": "Please provide signature"
+      },
+    },
+    "phone_hint": "Please type in valid phone number",
+    "fax_hint": "Please type in valid fax number"
+  },
+
```

Comment:
As I mentioned, I did not bother trying to figure which pieces of the translation could be reused. I just copied the original translations strings and updated them as needed.